packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -178,6 +178,7 @@ class BottomNavigationBar extends StatefulWidget {
     this.backgroundColor,
     this.iconSize = 24.0,
     Color selectedItemColor,
+    this.activeBackgroundColor = Colors.transparent,
```

Comment:
Is there a reason why this is a default value of Colors.transparent, but CupertinoTabBar's is not?

You may want to assert here as well.

Reply:
its `transparent` now for both variants